bin/deploy.rb

```diff
@@ -271,6 +299,22 @@ def print_newlines(n: 3)
   def check_current_git_branch
     execute_safely('git rev-parse --abbrev-ref HEAD').strip
   end
+
+  def notification_text
+    <<~NOTIFICATION
+      #{notification_emoji} Deployed `simple-server` `#{tag_to_deploy}` to #{country_to_deploy} #{current_environment}
+      #{changelog}
+    NOTIFICATION
+  end
+
+  def notification_emoji
```

Comment:
Want to stick this as a constant upstairs and freeze it and then just reference it in `notification_text`?

Reply:
Addressed in b755f4f